packages/schema-extrct/tests/generics/classes.spec.ts

```diff
@@ -30,46 +30,47 @@ describe('schema-extrct - generic classes',()=>{
                 "typeof MyClass" : {
                     "$ref":ClassConstructorSchemaId,
                     "genericParams": [{
-                        "name":"T"
-                    },{
                         "name":"P"
+                    },{
+                        "name":"T"
                     }],
                     "arguments":[
                         {
-                            "$ref":"#typeof MyClass!T",
+                            "$ref":"#MyClass!T",
```

Comment:
This change could be a mistake, this ref seems a bit confusing so I changed it but I am not sure if the constructer arguments should be "#MyClass!T" or "#typeof MyClass!T".